net-analyzer/openvas-scanner/openvas-scanner-5.1.3.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR="emake"
+inherit cmake-utils systemd
+MY_PN="openvas-scanner"
+
+DESCRIPTION="A remote security scanner for Linux (OpenVAS-scanner)"
+HOMEPAGE="http://www.openvas.org/"
+SRC_URI="https://github.com/greenbone/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86"
+IUSE="+extras"
+
+DEPEND="
+	dev-db/redis
+	dev-libs/libgcrypt:0=
+	>=net-analyzer/openvas-libraries-9.0.3
+	net-libs/gnutls:=[tools]
+	net-libs/libssh:="
+
+RDEPEND="
+	${DEPEND}
+	!net-analyzer/openvas-tools"
+
+BDEPEND="
+	extras? ( app-doc/doxygen[dot] )
```

Comment:
minor styling thing, but conditionals should come after unconditionals. I saw a 2nd ebuild with the same BDEPEND list, don't remember which one it was.

Reply:
thanks for the tip.fixed all conditionals | unconditionals sortings